contrib/clemux/docker/debile-slave/Dockerfile

```diff
@@ -15,7 +15,16 @@ COPY *.deb /tmp/
 
 WORKDIR /tmp
 
-RUN apt-get update && apt-get install -y python python-dput dput-ng python-firehose adduser python-debian python-requests python-yaml sbuild python-schroot
+RUN apt-get update && apt-get install -y python python-dput dput-ng python-firehose adduser python-debian python-requests python-yaml python-schroot wget
+
+# Use a custom version of sbuild written by Léo Cavaillé that adds an
+# option --use-schroot-session, which enables starting sbuild with an
+# existing schroot session.
+RUN wget https://dimjasevic.net/marko/debile/libsbuild-perl_0.69.0-2_all.deb
```

Comment:
I can do that, but it will significantly increase the repo size, and I was also hoping we would push the patch upstream (or at least to the sbuild Debian source package), have it merged, and then these wget lines would be obsolete.